src/GraphQL/Schemas/Primary/Substrate/Mutations/BurnMutation.php

```diff
@@ -125,7 +125,7 @@ protected function rulesWithValidation(array $args): array
         $min = Arr::get($args, 'params.removeTokenStorage', false) ? 0 : 1;
 
         return [
-            'collectionId' => [$removeTokenStorage ? new IsCollectionOwner() : 'exists:collections,collection_chain_id'],
+            'collectionId' => [$min ? new IsCollectionOwner() : 'exists:collections,collection_chain_id'],
```

Comment:
```suggestion
            'collectionId' => [$min == 0 ? new IsCollectionOwner() : 'exists:collections,collection_chain_id'],
```
Is this logic correct?  I take it if `$min` is 0 then we want to validate the collection owner, as only they can remove storage, however 0 would evaluate as `false` here wouldn't it?